Update domain-model.md

BeateRixen opened the pull request.

BeateRixen committed:
Update domain-model.md

BeateRixen commented on content/concepts/domain-model.md:
```diff
@@ -95,7 +95,7 @@ While designing the data models, consider the following:
 Fragments use a naming convention to avoid conflicts between different parties supplying fragment information, similar to Java or other programming languages.
 
-In the example above, `c8y_Position` is a combination of "c8y" (a shorthand for "Cumulocity"), an underscore and "Position". Together they form a set of standard fragments. Fragment definitions can be found in the [fragment library](/device-integration/fragment-library/).
+In the example above, `c8y_Position` is a combination of "c8y" (a shorthand for "Cumulocity"), an underscore and "Position". Together they form a set of standard fragments, see also [Fragment library](/device-integration/fragment-library/).
```
@l2p- I suggest to either remove the sentence "Fragment definitions can be found in the [fragment library](/device-integration/fragment-library/)" here entirely or to keep it as a generic link. We anyway mention the fragment library some lines above (line 54).

aaronraab replied:
Why do we use a capital letter in the first noun. I would either expect fragment library or Fragment Library but not Fragment library?

aaronraab replied:
I also don't think we need to provide the reference link again.

BeateRixen replied:
> Why do we use a capital letter in the first noun. I would either expect fragment library or Fragment Library but not Fragment library?

Because it is a link to the section in the documentation that has the heading "Fragment library". Probably more obvious if you look at the output, which shows a link here.

l2p- commented on content/concepts/domain-model.md:
```diff
@@ -95,7 +95,7 @@ While designing the data models, consider the following:
 Fragments use a naming convention to avoid conflicts between different parties supplying fragment information, similar to Java or other programming languages.
 
-In the example above, `c8y_Position` is a combination of "c8y" (a shorthand for "Cumulocity"), an underscore and "Position". Together they form a set of standard fragments. Fragment definitions can be found in the [fragment library](/device-integration/fragment-library/).
+In the example above, `c8y_Position` is a combination of "c8y" (a shorthand for "Cumulocity"), an underscore and "Position". Together they form a set of standard fragments, see also [Fragment library](/device-integration/fragment-library/).
```
There is no "set of standard fragments". Fragment library is documentation about how to make a standard set of DM functionalities work with devices.
```suggestion
In the example above, `c8y_Position` is a combination of "c8y" (a shorthand for "Cumulocity"), an underscore and "Position". For more information about various Device Management functionalities and their associated fragments, see also [Fragment library](/device-integration/fragment-library/).
```

l2p- reviewed (commented):
see suggestion

BeateRixen committed:
Update content/concepts/domain-model.md

Co-authored-by: Philipp Emmel <philipp.emmel@softwareag.com>

BeateRixen commented on content/concepts/domain-model.md:
```diff
@@ -51,7 +51,7 @@ Using this approach, the modeling devices can make a difference between modeling
 The approach also enables developing generic application components. For example, as soon as a managed object has a position fragment (`c8y_Position`), it can be placed on a map. As soon as it has a relay (`c8y_Relay`), it can be switched on and off using the respective device control command as described below.
 
-For more information on fragments and how managed objects are structured, see [Fragment library](/device-integration/fragment-library/).
+For more information on fragments and its usage, see the [Fragment library](/device-integration/fragment-library/).
```
```suggestion
For more information on fragments and their usage, see the [Fragment library](/device-integration/fragment-library/).
```

BeateRixen committed:
Update content/concepts/domain-model.md

BeateRixen committed:
Update domain-model.md

BeateRixen commented:
@l2p- @aaronraab I moved the updated text for the link to the fragments library to the general section on fragments and removed it from the section on naming conventions. Please approve.
